layouts/partials/footer/mermaid.html

```diff
@@ -0,0 +1,15 @@
+{{- $defaultMermaid := .Site.Data.config.default.mermaid -}}
+{{- $mermaid := ( or .Site.Data.config.mermaid $defaultMermaid ) -}}
+
+{{- if eq .Params.mermaid true -}}
+
+<script src='{{- (or $mermaid.library.path $defaultMermaid.library.path) -}}' />
```

Comment:
`{{- (or $mermaid.library.path $defaultMermaid.library.path) -}}` is not necessary. If custom config is present, everything should be taken from that. If custom config is not present, everything should be taken from default config.

So just `{{ $mermaid.library.path }}` should do.

Because `$mermaid` is either custom config (if present) or the fallback default config.

( See, `katex.html` or `mathjax.html` )